estate/models/estate_property.py

```diff
@@ -0,0 +1,23 @@
+from odoo import fields, models
+
+class EstateProperty(models.Model):
```

Comment:
```suggestion
from odoo import fields, models


class EstateProperty(models.Model):
```

By convention and for readability, leave two blank lines between the imports and the class